en/developer_guide/block_override.rst

```diff
@@ -0,0 +1,117 @@
+Block Override
+==============
+
+Context
+-------
+
+From time to time, the block provided by Open Orchestra will fulfill some of your requirements
+but you would want to modify a little part of their behaviour.
+
+As the block is partially functional, you do not want to create an other block with the functionality.
+
+We provide you different ways to override part of the Open Orchestra block.
+
+Overloading the template of a block
+-----------------------------------
+
+Overloading the template of a block is achieved by creating a template in the following folder structure :
+
+.. code-block::
+
+    |_ app
+        |_ Resources
+            |_ {BUNDLE_NAME}
+                |_ views
+                    |_ {PATH_TEMPLATE}
+                        |_ {FILENAME}
+
+
+Example :
+~~~~~~~~~
+
+You need to override the template of the Content List Block block from the Back Office to display
+additional datas.
+
+.. code-block::
+
+    |_ Open-orchestra
+        |_ open-orchestra-cms-bundle
+            |_ BackofficeBundle
+                |_ Ressources
+                    |_ views
+                        |_ Block
+                            |_ ContentList
+                                |_ show.html.twig
+
+``Open-orchestra/open-orchestra-cms-bundle/BackofficeBundle/Ressources/views/Block/
+ContentList/show.html.twig``
+
+You simply create your template in next folder :
+
+.. code-block::
+
+    |_ app
+        |_ Resources
+            |_ OpenOrchestraBackofficeBundle
+                |_ views
+                    |_ Block
+                        |_ ContentList
+                            |_ show.html.twig
+
+``app/Ressources/OpenOrchestraBackofficeBundle/views/Block/ContentList/show.html.twig``
+
+See again `symfony2 documentation`_
+
+Overloading directly a block
+----------------------------
+
+As an introduction to this part, I would suggest you to be familiar with the `block parameter`_
+documentation page.
+
+Open Orchestra gives you two ways to override a block.
```

Comment:
The further descriptions tell how to do but don't indicate how to choose one or the other solution.
I think some use cases should be appreciated.